addons/binding/org.openhab.binding.globalcache/src/main/java/org/openhab/binding/globalcache/handler/GlobalCacheHandler.java

```diff
@@ -66,14 +71,17 @@
 
     private InetAddress ifAddress;
     private CommandProcessor commandProcessor;
-    ScheduledExecutorService scheduledExecutorService = Executors.newScheduledThreadPool(MAX_GC_DEVICES);
-    ScheduledFuture<?> scheduledFuture;
+    private ScheduledExecutorService scheduledExecutorService = Executors.newScheduledThreadPool(MAX_GC_THREADS);
```

Comment:
I wasn't aware I could use this to get my own dedicated pool.  Thanks!

I'll rework this.  And, yes, 5 should be enough (command processor, serial port 1 reader, serial port 2 reader, and connection monitor threads).  Ok to do the rework in this PR?

I'll need to do this in one of my other bindings, as well.